qcengine/programs/mrchem.py

```diff
@@ -132,6 +133,10 @@ def compute(self, input_model: "AtomicInput", config: "TaskConfig") -> "AtomicRe
                 mrchem_output = mrchem_json["output"]
                 output_data["success"] = mrchem_output["success"]
                 output_data["provenance"] = mrchem_output["provenance"]
+                # parallel set up
+                output_data["provenance"].update(job_input["parallel"])
+                # remove nthreads, above info is more detailed anyway
+                output_data["provenance"].pop("nthreads")
```

Comment:
It's troubled me since I first read it that `nthreads` gets removed. Even though `total_cores` added below is the more detailed counterpart, people may expect the field. Do you think it should (a) be left to un-detailed value, (b) have `total_cores` copied in, or (c) definitely remove the field?

Reply:
Let's add for now, pending trouble.

Reply:
Hei Lori, sorry for not replying earlier. The reason to drop it was that the information in `job_input["parallel"]` reflects more accurately how the MPI ranks and OpenMP threads were distributed for the computation. MRChem allocates these resources a bit peculiarly...